addons/ui/org.openhab.ui.cometvisu/src/main/java/org/openhab/ui/cometvisu/backend/ChartResource.java

```diff
@@ -237,7 +237,7 @@ public Object getRrdSeries(QueryablePersistenceService persistenceService, Item
             logger.debug("no rrd file found '{}'", (RRD_FOLDER + File.separator + item.getName() + ".rrd"));
             return getPersistenceSeries(persistenceService, item, timeBegin, timeEnd, resolution);
         } catch (Exception e) {
-            logger.error(e.getLocalizedMessage() + ": fallback to generic persistance service");
+            logger.error("{}: fallback to generic persistance service", e.getLocalizedMessage());
```

Comment:
I would log the exception instead, it is an error, I want to be able to debug the stacktrace